master/internal/configpolicy/task_config_policy_intg_test.go

```diff
@@ -328,6 +395,19 @@ func addConstraints(t *testing.T, user model.User, wkspID *int, constraints stri
 	require.NoError(t, err)
 }
 
+func addConfigs(t *testing.T, user model.User, wkspID *int, configs string, workloadType string) {
```

Comment:
nit: How come "configs" is plural everywhere here?

Reply:
No reason. I can change it to "config".